sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <full-page-container>
+    <h2 class="color-blue">Search Institution</h2>
+    <v-row class="mt-5">
+      <v-col>
+        <div class="p-fluid p-formgrid p-grid">
+          <div class="p-field p-col-12 p-md-4">
+            <label class="field-required" for="legalName">Legal Name</label>
+            <InputText name="legalName" v-model="legalName" />
+          </div>
+          <div class="p-field p-col-12 p-md-4">
+            <label class="field-required" for="operatingName"
+              >Operating Name</label
+            >
+            <InputText name="operatingName" v-model="operatingName" />
+          </div></div
+      ></v-col>
+      <v-col class="mt-9" cols="auto"
+        ><v-btn
+          :disabled="!legalName && !operatingName"
+          color="primary"
+          @click="searchInstitutions"
+          >Search</v-btn
+        ></v-col
+      >
+    </v-row>
+
+    <DataTable
+      v-if="institutionsFound"
+      class="mt-4"
+      :autoLayout="true"
+      :value="institutions"
+    >
+      <Column field="legalName" header="Legal Name" :sortable="true">
+        <template #body="slotProps">
+          <div class="p-text-capitalize">
+            {{ slotProps.data.legalName }}
+          </div>
+        </template>
+      </Column>
+      <Column field="operatingName" header="Operating Name" :sortable="true">
+        <template #body="slotProps">
+          <div class="p-text-capitalize">
+            {{ slotProps.data.operatingName }}
+          </div>
+        </template>
+      </Column>
+      <Column field="address" header="Address">
+        <template #body="slotProps">
+          <div
+            v-for="addressLineItem in addressLineItemList(
```

Comment:
As we talked, I do not think that this is the way to go. It seems that we are overcomplicating something that is supposed to be simple.